src/Functions/coverage.cpp

```diff
@@ -6,6 +6,7 @@
 #include <Columns/ColumnVector.h>
 #include <Columns/ColumnsNumber.h>
 #include <Columns/ColumnConst.h>
+#include <Columns/ColumnsNumber.h>
```

Comment:
I think this is the result of merging master, probably I did something wrong. Will fix in another PR.